app/helpers/application_helper.rb

```diff
@@ -1,2 +1,128 @@
 module ApplicationHelper
+
+  def recreation_list
+    [
+      [' '],
+      ['Ski'],
+      ['Board'],
+      ['Both'],
+      ['Other']
+    ]
+  end
+
+  def trip_list
+    [
+      [' '],
+      ['One Way'],
+      ['Round Trip']
+    ]
+  end
+
+  def pass_list
+    [
+      [' '],
+      ['IKON'],
+      ['Epic'],
+      ['Other'],
+      ['No Preference'],
+      ['No Pass']
+    ]
+  end
+
+  def dates_list
+    [
+      [' '],
+      ['Monday'],
+      ['Tuesday'],
+      ['Wednesday'],
+      ['Thursday'],
+      ['Friday'],
+      ['Saturday'],
+      ['Sunday'],
+      ['Weekend Fri-Sun'],
+      ['Any Day']
+    ]
+  end
+
+  def days_list
+    [
+      [' '],
+      ['01'],
+      ['02'],
+      ['03'],
+      ['04'],
+      ['05'],
+      ['06'],
+      ['07'],
+      ['08'],
+      ['09'],
+      ['10'],
+      ['11'],
+      ['12'],
+      ['13'],
+      ['14'],
+      ['15'],
+      ['16'],
+      ['17'],
+      ['18'],
+      ['19'],
+      ['20'],
+      ['21'],
+      ['22'],
+      ['23'],
+      ['24'],
+      ['25'],
+      ['26'],
+      ['27'],
+      ['28'],
+      ['29'],
+      ['30'],
+      ['31']
```

Comment:
Okay, this may be a pipe dream, but is there a way for us to put month first and then limit the day options by the number of days in that month? Feel free to tell me I'm going too far with this. I definitely don't think it has to be part of MVC, but it would be a nice addition to this I think.

Reply:
Super agree! I thought about doing some conditionals in there, but am thinking for MVP, having a check that they entered a valid date may be an easier option

Reply:
Ah yeah good call. I think valid date works for MVP.